resources/views/components/page-headers/container.blade.php

```diff
@@ -0,0 +1,58 @@
+@props([
+    'label',
+    'breakpoint' => 'md',
+    'extra' => null,
+])
+
+@php
+    $wrapperBreakpointClass = [
+        'sm' => 'sm:items-center sm:rounded-lg sm:border sm:border-theme-secondary-300 sm:dark:border-theme-secondary-800',
+        'md' => 'md:items-center md:rounded-lg md:border md:border-theme-secondary-300 md:dark:border-theme-secondary-800',
+    ][$breakpoint] ?? 'md:items-center md:rounded-lg md:border md:border-theme-secondary-300 md:dark:border-theme-secondary-800';
+
+    $detailBreakpointClass = [
+        'sm' => 'sm:flex-row sm:items-center sm:space-y-0 sm:space-x-3',
+        'md' => 'md:flex-row md:items-center md:space-y-0 md:space-x-3',
+    ][$breakpoint] ?? 'md:flex-row md:items-center md:space-y-0 md:space-x-3';
+
+    $labelBreakpointClass = [
+        'sm' => 'sm:px-4 sm:py-[14.5px] sm:bg-theme-secondary-200 sm:dark:bg-black sm:text-base md:text-lg sm:!leading-[21px]',
```

Comment:
can you move the leading class to tailwind config

Reply:
I'll make the card generic in that it will add all instances of `leading-[]` and might as well do `width-[]`, if they are used more than once